src/core/lib/iomgr/tcp_client_windows.cc

```diff
@@ -194,9 +219,9 @@ static int64_t tcp_connect(grpc_closure* on_done, grpc_endpoint** endpoint,
 
   socket = grpc_winsocket_create(sock, "client");
   info = &socket->write_info;
+
```

Comment:
nit: for consistency with the codebase, please remove blank lines from inside function bodies.